### PR TITLE
Renaming to Zingo and ZingoMobile. IOS.

### PR DESCRIPTION
**Pull request by juanky201271**
I change all the old references to Zingo and ZingoMobile and I change the name of the wallet file inside of IOS to wallet, the same in both envs.

I can't test it in my ios laptop for now, but I will soon.

### CODE REVIEW

File: ios/RPCModule.m [hunk 1]
```diff
@@ -28,7 +28,7 @@ @implementation RPCModule
   NSString *documentsDirectory = [paths objectAtIndex:0];
 
   // Write to user's documents app directory
-  NSString *fileName = [NSString stringWithFormat:@"%@/zecwallet-lite.dat.txt",
+  NSString *fileName = [NSString stringWithFormat:@"%@/wallet.dat.txt",
```
Comment (skyl) [inline ios/RPCModule.m]: so then this file needs to be renamed too? Everything that I didn't change needed a filename change as well, as far as I tried ..

Comment (skyl) [reply ios/RPCModule.m]: ah on second thought, I think maybe this is the file that the program is going to create maybe - not one that is checked in so it's fine.

**Review by skyl — DISMISSED** [review]
Not sure about all of these ios/ things TBH. utACK though - it runs for you `yarn ios`? `dev` branch runs for me right now so I can easily test after merge and we can just follow up if anything is weird

**Comment by juanky201271** [comment]
> Not sure about all of these ios/ things TBH. utACK though - it runs for you `yarn ios`? `dev` branch runs for me right now so I can easily test after merge and we can just follow up if anything is weird

I'm trying to set up my old mac, so I can't test it for now. Maybe, if you want, can test the ios app with my branch, or as you said merge it and test it after...

File: ios/RPCModule.m (hunk 1 above)
Comment (zancas) [reply ios/RPCModule.m]: To match `zingolib` this should be `zingo-wallet.dat.txt`

**Review by zancas — CHANGES_REQUESTED** [review]
Other than the wallet filename, these all look good to me.

File: ios/RPCModule.m (hunk 1 above)
Comment (juanky201271) [reply ios/RPCModule.m]: Sounds good to me, I'll do it right away.

Comment (juanky201271) [reply ios/RPCModule.m]: done! I hope the `-` don't break something in the ios env.

**Review by zancas — APPROVED** [review]

**Pull request merged**